src/components/TopErrorBoundary.tsx

```diff
@@ -81,8 +81,13 @@ export class TopErrorBoundary extends React.Component<
             {t("errorSplash.clearCanvasMessage")}
             <button
               onClick={() => {
-                localStorage.clear();
-                window.location.reload();
+                try {
+                  localStorage.clear();
+                } catch (err) {
+                  console.error(err);
+                } finally {
+                  window.location.reload();
+                }
```

Comment:
It doesn't make sense to reload on error. In fact it will increase the probability the error isn't logged to sentry.

Also please rename `err` → `error` (here and in your other changes) to align with rest of codebase.

Reply:
Other than this, I think it's good to go!